src/test/regress/sql/multi_extension.sql

```diff
@@ -112,6 +112,9 @@ ALTER EXTENSION citus UPDATE TO '9.0-1';
 ALTER EXTENSION citus UPDATE TO '9.0-2';
 ALTER EXTENSION citus UPDATE TO '9.1-1';
 ALTER EXTENSION citus UPDATE TO '9.2-1';
+ALTER EXTENSION citus UPDATE TO '9.2-2';
+ALTER EXTENSION citus UPDATE TO '9.3-1';
```

Comment:
```suggestion
/*
 * As we mistekanly bumped version to 9.3.1 (in previous release),
 * we support updating citus from 9.3.1 to 9.2.4, but we do not
 * support explicitly updating it to to 9.3.1.
 * Hence below fail update (to 9.3.1) command should fail.
 */
ALTER EXTENSION citus UPDATE TO '9.3-1';
```

Reply:
We better stick to `9.x-y` format in these comments, as they refer to migration files and not the versions